### DISCUSSION TITLE
Missing Inventory-part of Invoice after upgrade from 5.2 to 6.1 - Any idea/help?

### DISCUSSION THREAD

**Discussion by mad99** [YetiForceCompany/YetiForceCRM#14156] [General]
Hi,
maybe I'm looking on the total wrong palces. But I can't figure out, how I can use the invoice module in 6.1.
Today I have upgraded the system from 5.2 => 5.3 => 6.0 => 6.1 => 6.1 SP v6.0.

All seamed to be fine. But in the invoice part I can't find the inventory-part of the invoice. The right part with all the details about number of products, price, vat. All is gone. I search at different places, also in browser's source-code, on other parts of the CRM, tried to find some settings for invoice (but didn't find any that would match), tried with 3 different browsers, also with Dev-Tools open so the browser-cache will be flushed.

- Maybe this part I have to use with a new button for? 
- Or do I have to use this part from an other path then «Accounting /Sales invoices / Edit» is this moved to an other part of the CRM?
- Or do I have to update some part of the Database/Cache after Upgrade? ( The values from all the inventory-part of all the invoices are still in the database. I can also print the right invoices with all the details in it. But in the Edit-View there are no inventory-details to find.

**- Does anybody has the same problem and a solution?**
**- Or some hint or help?**

In YF 5.2 the invoice-inventory-part is on the right side: all is fine! 
![YetiForce Edit-Invoice 5 2](https://user-images.githubusercontent.com/6169308/106632706-59b12b80-657e-11eb-86e9-1427d727faa6.jpg)
In YF 6.1 the invoice-inventory-part is missing on the right side and also everywhere.
![YetiForce Edit-Invoice 6 1](https://user-images.githubusercontent.com/6169308/106632920-8f561480-657e-11eb-92a7-96af8fc90d95.jpg)

**Comment by mariuszkrzaczkowski** [comment]
first key question: Were all updates successful?

second question: Do you have the correct server configuration

**Comment by mad99** [comment]
Thank you for your reply, @mariuszkrzaczkowski 

First: Yes there was always a popup «the upgrade was succsessfull» (or a similar text).

Second: 5.2 YetiForce was fine with the same server confguration. I did not change configurations.

There are no other problems within the 6.1 YetiForce, I guess. All other pages seams to be fine. Only the invoice right-side is missing.
But I guess also in 6.1 the inventory should be shown on the right side, not somewhere else - is this correct?

But back to the settings. Are there maybe some server configurations which is/are absolutely critical for 6.1 or could one of them be the obvious reason for the missing invoice part and the same setting in the past in 5.2 has no problem triggered?

**Comment by mad99** [comment]
Crazy, I did the missing «encryption» part and as the last try in 5.2 after the cronjob, I could NOT login. Tried also the PasswordReset.php workaround. No way!!

During the Upgrade I backuped each step with file and database. So I reset the database, but no login. 
Then I deleted the file-system and rebuild it from backup, also I did a DB-reset, and stopped the cron (for stop the encryption process «play-again»).

_After this I could login again. Great!
**_But mutch greater => The inventory in the invoice-right-part is appeared! I don't know why or how. But it is working now, with the same Server-Config._**

So I guess, the Problem (I don't know what kind of) is solved.

## Off-Topic: 

- Are there some steps after YetiForce Encryption Process, that I have to change, so I could login like before. With same username and password? 

What could be the reason, so I can't login after the encryption process?

**Reply by vovpff** [reply]
@mad99 After update all your users should clean browser cache.
Also I has checked what I has problems with login to my updated 6.1 and gitdev also. Solution is to flush all cookies for crm websites.

**Reply by mad99** [reply]
Thanks @vovpff,
Will try it again, and flush «all» cache (I tried with open Dev-Tool, what should do the same) or try it with anonymous-window.

**Comment by Giorgos-Hellene** [comment]
Try to select advanced on Edit->sales invoices. It might has selected Basic now.
Wait for some minutes until run the cron and check it

![εικόνα](https://user-images.githubusercontent.com/50912403/107353245-81037d80-6ad5-11eb-8cf6-a2b99a7cf607.png)

**Comment by mad99** [comment]
Hey @Giorgos-Hellene,
Great to know, if the part will be hidden again. 
Then I will come back to this topic and your tip and try it. 
Currently the part is visible. Without knowing why ;..o).
Thanks
